tt_metal/hw/firmware/src/brisc.cc

```diff
@@ -136,12 +152,21 @@ void enable_power_management() {
 void set_deassert_addresses() {
     volatile tt_reg_ptr uint32_t* cfg_regs = core.cfg_regs_base(0);
 
+#ifdef ARCH_BLACKHOLE
+    WRITE_REG(RISCV_DEBUG_REG_NCRISC_RESET_PC, MEM_NCRISC_IRAM_BASE);
```

Comment:
The addresses `RISCV_DEBUG_REG_NCRISC_RESET_PC` don't exist in any file at the moment for whb0/gs, but they can be added to `tensix.h`, similar to blackhole, because it seems they share the same debug addresses (need to confirm with hardware team). As for why that specific NCRISC RESET PC is needed for blackhole, but not wormhole_b0, I am unsure. It was needed for blackhole runtime for buda. @ttmtrajkovic would you know which issue needed the above fix?